ls1intum/Artemis
# `Assessment`: Increase the maximum configurable length for complaints and complaint responses

### Checklist
#### General


#### Server
- [x] I documented the Java code.

#### Client

Closes: https://github.com/ls1intum/Artemis/issues/4664

### Motivation and Context


As discussed in https://github.com/ls1intum/Artemis/issues/4664 the maximum numbers of characters should be increased, as this is required for some instructors.

### Description

I've changed the column for complaints and complaints responses from `VARCHAR(5000)` to `TEXT` as this allows for up to 65535 characters per complaint. This is almost the amount excpected for a bachelors thesis, so this will very likely will be enough for every use-case.

I've also updated the contstants (introduced in https://github.com/ls1intum/Artemis/pull/4810) for the maximum value for those texts in Front- and Backend.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student

1) Log in to Artemis
2) Navigate to Course Administration
3) Increase or decrease complaint text and complaint response text limit (e.g. 10000 and 20000)
4) File a complaint as a student
5) Response to that complaint as a tutor
6) Verify that these new limits work as expected


### Review Progress





#### Code Review
#### Manual Tests

Note: Actions fail due to lack of permissions; actual tests pass.
